CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ Mapbox welcomes participation and contributions from everyone.
 #### Features
 
 #### Bug fixes and improvements
+- Fixed reroute request interruption when setting the `NavigationRerouteController` [#5950](https://github.com/mapbox/mapbox-navigation-android/pull/5950).
```

Comment:
Let's make sure this is not causing regression before we cherry pick this commit. 
@RingerJK

Reply:
BTW release notes should be cherry-picked separately (if we finally decide to cherry-pick https://github.com/mapbox/mapbox-navigation-android/pull/5950).

Reply:
I believe most of these lines were cherry-picked by mistake.

Reply:
@Guardiola31337 do you mean that changelog should be fully cherry-picked from main?

Reply:
`CHANGELOG` is always going to bring conflicts when cherry-picking. What I meant was that those lines were not relevant to this cherry-pick (those are related to the other PR #5950 and should be cherry-picked separately - along with the rest of its changes).

Reply:
So, should there be a changelog entry for this change or not? I think there should be, but there isn't any at the moment.

Reply:
Yeah, for some reason it was removed. The only thing that I was saying was that that other entry was not for this PR but we should keep the one from this PR 👍

Reply:
@RingerJK you have added a wrong changelog entry.

Reply:
@Zayankovsky rebase issue, fixed 👍